drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -871,6 +871,7 @@ function DragAndDropBlock(runtime, element, configuration) {
                 input: input,
                 displayName: item.displayName,
                 imageURL: imageURL,
```

Comment:
Up to you. You can also just use `expandedImageURL` throughout the JS code. My main point was we didn't need two similar variables, when only one is actually used. But this is not something I feel strongly about.

Fallback for the old attribute name is not really important, because the old block was never properly expanding these URLs anyways, so the old attribute will never contain short URLs. It's fine to expand it just in case though.